tools/update-tester/src/testrunner.rs

```diff
@@ -339,6 +338,9 @@ impl TestClient {
             .into_iter()
             .flat_map(|tests| {
                 let mut client = connect_to(&root_config).0;
+                client
```

Comment:
Moving the code that sets the timezone to here instead of line 150 seems to have fixed the issue on my machine. It could also have been moved to run_test(), but then it would be called on every sql query and that seemed excessive.